anvio/kegg.py

```diff
@@ -2678,7 +2678,7 @@ def list_output_headers(self):
             desc_str = header_meta['description']
             type_str = header_meta['mode_type']
             mode_str = "output modes" if header_meta['mode_type'] == 'all' else "output mode"
-            self.run.info(header, f"{desc_str} [{type_str} {mode_str}]")
+            self.run.info(header, f"{desc_str} [{type_str} {mode_str}]", align_long_values=True)
```

Comment:
these seem to be unnecessary since the default value is set to `True` in the function header. Unless there is another reason I am missing to be explicit here.

Reply:
no reason! I put this there before I changed the default to `True`, and forgot to change it back. This is also now fixed.